docs/customization/api.rst

```diff
@@ -24,6 +24,16 @@ If your project was created before v1.10, make sure your API Platform config fol
         swagger:
             versions: [3]
 
+And if you want to modify serialization add this code to framework config:
+
+.. code-block:: yaml
+
+    # config/packages/framework.yaml
+    //...
+    serializer:
+        mapping:
+            paths: [ '%kernel.project_dir%/config/serialization' ]
+
```

Comment:
```suggestion
```

Reply:
Why remove? I have added the change with this to Sylius-standard but to master branch, still older versions need to have this config added manually.

Reply:
This note is also in `How to prepare project for customization?` section and we have information about supported version `If your project was created before v1.10...`

Reply:
just Symfony by default is looking for the `serializer` folder and then we would not need any configuration at all... But we may stay with this config to be closer to Api Platform